views/components/BarGraphTooltip/BarGraphTooltip.css

```diff
@@ -0,0 +1,26 @@
+.BarGraphTooltip {
+  width: 200px;
+  padding: 14px 14px 7px 14px;
+  background: rgba(255, 255, 255, 0.9);
+}
+.BarGraphTooltip_Header {
```

Comment:
should there be 2 underscores between the block and the element? That's part of the BEM convention, but not sure if our in-app convention is different. Also, this css file uses a capital letter for the element (Header) while the `ChartFilterForm` css file does not. Let's keep our conventions consistent 👍

Reply:
[094aa13](https://github.com/AMP-SCZ/dpdash/pull/529/commits/094aa13a95831d13d66ca3fc781ced6dbe96a4c7)
We use one underscore and good catch on the uppercase letters